### PR TITLE
Revise README

### PR DESCRIPTION
**Pull request by oOKexOo**
**When merged this pull request will:**
- Update the contend in the README file. This revision will be applied to Steam Workshop and the BI forums as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,86 +34,134 @@
...
+- [Reporting Issues, Requesting Features and Changes](#reporting-issues,-requesting-features-and-changes)
+- [How to Contribute](#how-to-contribute)
+- [Authors](#authors)
+- [Contact](#contact)
+- [License](#license)
+
+## Features
+- Visual changes to the Zeus interface
+- Tons of new modules:
+    - ACE Medical (_e.g._ heal, injury)
```
Comment (CreepPork) [inline README.md]: Needs info that it works for default medical as well.

File: README.md [hunk 2]
```diff
@@ -34,86 +34,134 @@
...
+    - AI Behaviour (_e.g._ animations, garrison buildings, surrender)
+    - Arsenal (add/modify)
```
Comment (CreepPork) [inline README.md]: Everywhere there is e.g. but here there isn't any? Style!

File: README.md [hunk 3]
```diff
@@ -34,86 +34,134 @@
...
+    - Buildings (_e.g._ destruction, breachable doors)
+    - Dev Tools (_e.g._ execute code, config/function viewer)
+    - Environment (_e.g._ advanced weather, earthquakes)
+    - Equipment (_e.g._ toggle tac light)
+    - Fire Support (_e.g._ suppressive fire)
+    - Objects (_e.g._ attach to, toggle simulation, IEDs)
+    - Reinforcments (spawn reinforcements on the fly)
```
Comment (CreepPork) [inline README.md]: Same here

Comment (CreepPork) [reply README.md]: Maybe not so applicable

File: README.md [hunk 4]
```diff
@@ -34,86 +34,134 @@
...
+    - Copy/past units including the full loadouts
+    - Force unit to pop smoke
+    - _etc._
+- Customizations:
+    - Settings (e.g. faction filter)
+
+## Language Localization
+Achilles is available for the following languages:
+- English
+- French (rather poor)
```
Comment (CreepPork) [inline README.md]: Remove the rather poor, doesn't matter. Makes the addon seem unprofessional.

File: README.md [hunk 5]
```diff
@@ -34,86 +34,134 @@
...
+- German
+- Japanese
+- Russian
+- Simplified Chinese
+- Traditional Chinese
+
+If your language is not on the list, the english localization will be used. New translations are welcome (check out [How to Contribute](##how-to-contribute)).
```
Comment (CreepPork) [inline README.md]: Captial English

File: README.md [hunk 6]
```diff
@@ -34,86 +34,134 @@
...
-## Authors
+### Add a New Module
+A good starting point for your new module is the [custom module framework](https://github.com/ArmaAchilles/Achilles/wiki/Custom-Modules).
+We will gladly help you to port your custom module to Achilles (message us on [our Discord server](https://discord.gg/kN7Jnhr)).
 
-- **[Anton Struyk](https://github.com/astruyk)** - [Ares](https://github.com/astruyk/Ares)
-- **[Kex](https://github.com/oOKexOo)** - Achilles
-- **[CreepPork](https://github.com/CreepPork)** - Achilles
+### Add a Translation
+You will have to edit the [stringtable.xml](https://github.com/ArmaAchilles/Achilles/blob/master/%40AresModAchillesExpansion/addons/language_f/stringtable.xml). The basics about string tables can be found on the [Bohemia Interactive Wiki](https://community.bistudio.com/wiki/Stringtable.xml). If you don't want to edit the file with a text editor, you can use a dedicated tool such as [Tabler](http://www.armaholic.com/page.php?id=26320).
```
Comment (CreepPork) [inline README.md]: _You will_ seems quite forceful.

File: README.md [hunk 7]
```diff
@@ -34,86 +34,134 @@
...
+- [Zeus/Eden Compositions for CUP (ZECCUP)](https://github.com/LISTINGS09/ZECCUP)
+- [Zeus/Eden Interiors (ZEI)](https://github.com/LISTINGS09/ZEI)
 
-To install Achilles you can download it from:
-- [GitHub](https://github.com/ArmaAchilles/AresModAchillesExpansion/releases)
-- [Steam Workshop](http://steamcommunity.com/sharedfiles/filedetails/?id=723217262)
-- [Play with Six](http://withsix.com/p/Arma-3/mods/mc1X_8GXVEG0STlO_DWjNA/Ares-Mod-Achilles-Expansion)
-- [Armaholic](http://www.armaholic.com/page.php?id=31235)
+## Documentation
+Check out the [Achilles Wiki](https://github.com/ArmaAchilles/Achilles/wiki) and the [Achilles Tutorial Series on Youtube](https://www.youtube.com/watch?v=qjD2GX9rCA4&list=PL7del_lBYPTTNEmfPfzKVHxRx8Vx8DxHg). The documentation is rather limited at the time and thus, you will certainly have questions. We will gladly answer them on [our Discord server](https://discord.gg/kN7Jnhr).
```
Comment (CreepPork) [inline README.md]: YouTube

File: README.md [hunk 8]
```diff
@@ -34,86 +34,134 @@
...
+- [License](#license)
+
+## Features
+- Visual changes to the Zeus interface
+- Tons of new modules:
+    - ACE Medical (_e.g._ heal, injury)
+    - AI Behaviour (_e.g._ animations, garrison buildings, surrender)
+    - Arsenal (add/modify)
+    - Buildings (_e.g._ destruction, breachable doors)
+    - Dev Tools (_e.g._ execute code, config/function viewer)
```
Comment (CreepPork) [inline README.md]: Development Tools

File: README.md [hunk 9]
```diff
@@ -78,7 +77,7 @@
     - _e.g._ seek and destroy, paradrop, sling load
 - Hotkeys:
     - Copy/past units including the full loadouts
-    - Force unit to pop smoke
+    - Remote control
```
Comment (CreepPork) [inline README.md]: Remote Control

File: README.md [hunk 10]
```diff
@@ -93,7 +92,7 @@ Achilles is available for the following languages:
 - Simplified Chinese
 - Traditional Chinese
 
-If your language is not on the list, the English localization will be used. New translations are welcome (check out [How to Contribute](##how-to-contribute)).
+If your language is not on the list, the English localization will be used. New translations are welcome (check out [How to Contribute](#how-to-contribute)).
```
Comment (CreepPork) [inline README.md]: We welcome any new translations!

File: README.md [hunk 11]
```diff
@@ -144,19 +143,19 @@ We always welcome contribution to the repository and thank you for your work! We
...
 - [Basic Steps](#basic-steps)
-- [Setting up Your Development Environment](#setting-up-your-local-development-environment)
+- [Setting up Your Local Development Environment](#setting-up-your-local-development-environment)
 - [Add a New Module](#add-a-new-module)
 - [Add a Translation](#add-a-translation)
 
 ### Basic Steps
 1. [Fork the Achilles repository](https://github.com/ArmaAchilles/Achilles/fork)
-2. [Set up your development environment](#setting-up-your-local-development-environment)
+2. [Set up your local development environment](#setting-up-your-local-development-environment)
```
Comment (CreepPork) [inline README.md]: Why not capitalized here?

Comment (oOKexOo) [reply README.md]: It's not a title

File: README.md (hunk 9 above)
Comment (oOKexOo) [reply README.md]: Also not a title

File: README.md (hunk 10 above)
Comment (oOKexOo) [reply README.md]: Agreed

**Comment by oOKexOo** [comment]
It makes more sense to store that `README.steamworkshop` in the repo than just on my local machine.

File: README.md (hunk 9 above)
Comment (CreepPork) [reply README.md]: Looks better in this instance.

**Pull request merged**